From 3ade16f51cccd661d773ca053da326bbd83d7d25 Mon Sep 17 00:00:00 2001
From: "Douglas A. Augusto" <daaugusto@gmail.com>
Date: Tue, 6 Feb 2018 15:24:59 -0200
Subject: [PATCH] Integrates Ingo Heimbach's OSX compatibility changes. Thanks.

---
 LanguageIdentifier.sh | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/LanguageIdentifier.sh b/LanguageIdentifier.sh
index 00f04fc..3f9f835 100755
--- a/LanguageIdentifier.sh
+++ b/LanguageIdentifier.sh
@@ -80,7 +80,7 @@ NLANGS=1
 #    af ar az be bg br bs ca cs cy da de el en eo eo-h eo-x es et eu fi fr ga
 #    he hi hr hu hy is it ja la lt lv nl no pl pt-br pt-pt ro ru sk sl sq sr sv
 #    sw ta th tl tr ua vi zh
-SUBS=":"   # sed ':' does nothing (same as cat)
+SUBS=""
 
 # Explicitly specify the file type
 FT=""
@@ -152,7 +152,7 @@ then
    # guessings | perform the substitutions (if any)| remove possible duplicate
    # languages | filter out undesired languages (matches whole line) | replace
    # EOL with comma
-   LANG="$(printf "%s\n" "$OUT" | awk '$1 !~ /^0.000/ {print $2}' | sed -E "$SUBS" | awk '!a[$0]++' | grep -E -x -m $NLANGS "$LANGS" | paste -sd',')"
+   LANG="$(printf "%s\n" "$OUT" | awk '$1 !~ /^0.000/ {print $2}' | sed -E "$SUBS" | awk '!a[$0]++' | grep -E -x -m $NLANGS "$LANGS" | paste -sd',' -)"
 
    # Output guessed language (empty if it could not identify)
    printf "%s" "$LANG"
